contacts-app/src/layout/App.jsx

```diff
@@ -2,32 +2,49 @@ import React from "react";
 import FilterableContacts from "./../components/FilterableContacts/FilterableContacts";
 import NewContact from "./../components/NewContact/NewContact";
 import styles from "./App.module.css";
+import avatar1 from "./../avatar-1.png";
+import avatar2 from "./../avatar-2.png";
+import avatar3 from "./../avatar-3.png";
 
 class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      contactList: data
+      contactList: data,
+      invisible: false
     };
   }
 
   addNewContact = newContact => {
     this.setState({
-      contactList: [...this.state.contactList, newContact]
+      contactList: [...this.state.contactList, newContact],
+      invisible: false
+    });
+  };
+
+  showContent = event => {
```

Comment:
This is a creative approach to adding the page toggle! 
My only thought is how is it going to scale?